templates/api/dataset_loading/index.md

```diff
@@ -1,13 +1,18 @@
-# Dataset preprocessing
+# Dataset loading
```

Comment:
I wonder whether we should use `data_loading` as the name so as to avoid confusion with `keras.datasets`? WDYT?

Reply:
I think that's better. Made the change. Here's what the sidebar looks like http://screen/5aAs64Hzy6DUVJT

My main question is whether we should fold this into the keras utilities section... That would be more literal to where all the symbols are place, but might make this documentation less discoverable.

Reply:
You're right, semantically "data loading utilities" should be a section under "utilities". The discoverability argument is pretty significant though because right now `/api/preprocessing/image/` is a top-10 page, with more traffic that the Sequential docs, Model docs, the getting started page, etc. The setup in this PR looks good to me.